internal/storage/rocksdb/store.go

```diff
@@ -250,6 +252,38 @@ func (rdb *rocksDB) Close() error {
 	return nil
 }
 
+func (rdb *rocksDB) ReplaceDB(checkpointDir string) error {
+	backupDir := fmt.Sprintf("%s.bak", rdb.opts.folderName)
+	rdb.optimTrxnDB.Close()
+
+	err := storage.RenameFolder(rdb.opts.folderName, backupDir)
+	if err != nil {
+		rdb.opts.lgr.Error("Failed to backup existing db")
```

Comment:
Please include the error details in the error logs

Reply:
👍